src/BigQuery/BigQueryClient.php

```diff
@@ -194,8 +194,8 @@ public function __construct(array $config = [])
      *     @type bool $useQueryCache Whether to look for the result in the query
      *           cache.
      *     @type bool $useLegacySql Specifies whether to use BigQuery's legacy
-     *           SQL dialect for this query. **Defaults to** `true`. If set to
-     *           false, the query will use
+     *           SQL dialect for this query. **Defaults to** `false`. Unless set to
```

Comment:
> Unless set to true, the query will use [BigQuery's standard SQL](https://cloud.google.com/bigquery/sql-reference).

It could be just me, but this phrasing seems a little hard to parse. WDYT of something like the following?

> If set to true the query will use [BigQuery's legacy SQL](https://cloud.google.com/bigquery/docs/reference/legacy-sql), otherwise [BigQuery's standard SQL](https://cloud.google.com/bigquery/sql-reference).